Assets/Scripts/GhostController.cs

```diff
@@ -4,9 +4,13 @@
 
 public class GhostController : MonoBehaviour {
 	public float speed;
+    public Transform mainCameraPos;
```

Comment:
Indentation not good again... any many others, plz review the indentation stuff... LOL